front/pages/w/[wId]/assistant/gallery.tsx

```diff
@@ -263,8 +295,15 @@ export default function AssistantsGallery({
                   owner={owner}
                   plan={plan}
                   agentConfiguration={a}
-                  onShowDetails={() => {
-                    setShowDetails(a);
+                  onShowDetails={async () => {
+                    const href = {
+                      pathname: router.pathname,
+                      query: {
+                        ...router.query,
+                        assistantDetails: a.sId,
+                      },
+                    };
+                    await router.push(href);
```

Comment:
FWIW we won't leverage SSR on `<Link>` but I guess it's fine since the gallery already fetches all agent configurations with their authors. It's only missing the usage.

Reply:
Yep. I thiiiiink that's fine